takwimu/templates/takwimu/_includes/dataview/document.html

```diff
@@ -1,7 +1,13 @@
 {% load wagtailcore_tags wagtailembeds_tags %}
 
-{% embed value.document.file.url %}
-<iframe src="https://mozilla.github.io/pdf.js/web/viewer.html?file=
-    {{ HURUMAP.url }}{{ value.document.file.url }}"
-        frameborder="0" width="100%" height="500px">
-</iframe>
+<div class="card border-0">
+    <div class="card-body">
+        <div class="media">
```

Comment:
Why are we using card component?

Reply:
@DavidLemayian 
In order to have a consistence spacing for dataview components, they need to follow this component implementation 
https://github.com/TakwimuAfrica/Takwimu/blob/master/takwimu/templates/takwimu/_includes/dataview/entity_detail.html

Reply:
@mtuchi For spacing why not just use padding class?

Reply:
@DavidLemayian 
To reduce writing new css for the components i reused the `entity_detail.html` component since it already has nice spacing and `white background` from `card` class

Reply:
@mtuchi Just on two issues:

1. White background you have already implemented [here](https://github.com/TakwimuAfrica/Takwimu/pull/393/files#diff-67f1cbd6d0e7a9ceddecd15ac1e36d01R37)
2. Spacing would be achieved with *less HTML* and *no CSS* like so:
```
<div class="p-3">
{{ embed }}
</div>
```